### PR TITLE
Feedback Vienna

### PR DESCRIPTION
**Pull request by IObert**

### CODE REVIEW

File: exercises/05/readme.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ curl \
 
 For Windows users, this is the equivalent command (basically you have to use double quotes throughout, and therefore some must be escaped with `\`, and the line continuation character is `^` rather than `\`):
 
-```
+```sh
```
Comment (qmacro) [inline exercises/05/readme.md:217]: why the change from shell to sh (Q applies to other occurrences as well, of course :-))

File: exercises/05/readme.md [hunk 2]
```diff
@@ -223,11 +223,17 @@ curl ^
...
 :point_right: Order 9 copies of Life, The Universe And Everything (specifying an order ID):
 
-```
+```sh
 curl \
   -d '{"ID": "527ef85a-aef2-464b-89f6-6a3ce64f2e14", "book_ID":427,"quantity":9}' \
   -H 'Content-Type: application/json' \
   http://localhost:4004/catalog/Orders
+
+# For Windows users:
```
Comment (qmacro) [inline exercises/05/readme.md:232]: better as a separate non-code-formatted paragraph, as above, to keep the invocations separately in two code blocks (same comment for the rest of these same changes)

File: exercises/05/readme.md (hunk 1 above)
Comment (IObert) [reply exercises/05/readme.md:217]: I added it here because I started with 'sh' in the other places to stay consistent. Then, I noticed that we also use `bash` in other places and gave up :D 
I don't mind how we name it, as long as it is consistent. I agree. 
Which one do you prefer?

File: exercises/05/readme.md (hunk 2 above)
Comment (IObert) [reply exercises/05/readme.md:232]: ok

**Comment by maxstreifeneder** [comment]
merged, thanks!

File: exercises/05/readme.md (hunk 1 above)
Comment (IObert) [reply exercises/05/readme.md:217]: Fixes in PR #23
